src/tests/ftest/util/ior_test_base.py

```diff
@@ -166,16 +167,17 @@ def run_ior_with_pool(self, intercept=None, test_file_suffix="",
             self.dfuse = None
         return out
 
-    def update_ior_cmd_with_pool(self):
+    def update_ior_cmd_with_pool(self, create_cont=True):
         """Update ior_cmd with pool."""
         # Create a pool if one does not already exist
         if self.pool is None:
             self.create_pool()
         # Always create a container
```

Comment:
Now we don't always create a container, so we don't need this comment.

Reply:
Done